src/components/Organisms/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import './NavBarStyles.scss';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTicketAlt, faBars } from '@fortawesome/free-solid-svg-icons';
+import SearchBar from '../../Atoms/SearchBar';
+
+function NavBar() {
+  const [collapse, setCollapse] = useState();
+  const [toggleClass, setToggleClass] = useState('');
+  const [toggleAria, setToggleAria] = useState('');
+  const [navClass, setNavClass] = useState('');
+
+  const updateToggleClass = (isActive) => {
+    if (isActive) {
+      setToggleClass('navbar-toggler collapsed');
+      setToggleAria('false');
+      setNavClass('navbar-collapse collapse');
+      return;
+    }
+    setToggleClass('navbar-toggler');
+    setToggleAria('true');
+    setNavClass('navbar-collapse collapse show');
+  };
+
+  useEffect(() => {
+    const initialToggleValue = true;
+    setCollapse(initialToggleValue);
+    updateToggleClass(initialToggleValue);
+  }, []);
+
+  const toggleClick = () => {
+    const newToggleValue = !collapse;
+    setCollapse(newToggleValue);
+    updateToggleClass(newToggleValue);
+  };
+
+  return (
+    <nav className="navbar navbar-expand-lg o-nav-bar__filmstrip">
+      <NavLink
+        to="/"
+        className="o-nav-bar__ticket-logo"
+      >
+        <FontAwesomeIcon
+          className="fa-4x fas fa-ticket-alt mr-4"
+          icon={faTicketAlt}
+        />
+      </NavLink>
+      <button
+        className={toggleClass}
+        type="button"
+        data-toggle="collapse"
+        data-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent"
+        aria-expanded={toggleAria}
+        aria-label="Toggle navigation"
```

Comment:
I'd avoid the aria stuff unless completely necessary. It can guzzle time (I know it's in all the Bootstrap docs though, personally I usually remove it).